src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProjectSteps.java

```diff
@@ -0,0 +1,110 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.fundacionjala.core.selenium.WebDriverManager;
+import org.fundacionjala.pivotal.context.ProjectContext;
+import org.fundacionjala.pivotal.entities.Project;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.DashboardPage;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.ProjectPage;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.ProjectsSummaryPage;
+import org.fundacionjala.pivotal.ui.popups.CreateProjectPopup;
+import org.testng.asserts.SoftAssert;
+import static org.junit.Assert.assertTrue;
+import java.util.Map;
+
+public class ProjectSteps {
+
+    // Pages
+    private DashboardPage dashboardPage;
+    private CreateProjectPopup createProjectPopup;
+    private ProjectPage projectPage;
+    private ProjectsSummaryPage projectsSummaryPage;
+
+    // Entity
+    private Project project;
+
+    //Context
+    private final ProjectContext projectContext;
+
+    private static final int WAIT_TIME = 3000;
+    /**
+     * Adding Dependency injection to share Project Context information.
+     * @param sharedProjectContext
+     */
+    public ProjectSteps(final ProjectContext sharedProjectContext) {
+        this.projectContext = sharedProjectContext;
+    }
+
+    /**
+     * StepDef to open the Create Project pop-up.
```

Comment:
Avoid having StepDef in the javadoc, it can be only Opens the Create Project pop-up